src/main/java/frc/robot/commands/Drive.java

```diff
@@ -38,9 +38,9 @@ public void execute() {
     // Get Joystick inputs
     double xVelocity = xAxis.getAsDouble() * prefDrivetrain.driveSpeed.getValue();
```

Comment:
I think you maximum speed issue is here. Have you tried increasing this to a huge number? Also the value is in meters per second right? Shouldn't it be rotations per second? Unless it's converted under the hood